### ISSUE TITLE
std4docker vs sdthirlwall images

### ISSUE THREAD

**Issue by thomas-teknique** [sdt/docker-raspberry-pi-cross-compiler#19]
Hi

I was using sdt4docker/raspberry-pi-cross-compiler, but now I get a message saying "This image has been replaced by sdthirlwall/raspberry-pi-cross-compiler". I tried updating my image to the new cross compiler, but my own Dockerfiles are now broken, because they are expecting Ubuntu Trusty and not Debian Jessie. Can you keep sdt4docker/raspberry-pi-cross-compiler active for people who are using Dockerfiles expecting Trusty? Or is there a very compelling reason to use the new image including Jessie and change my Dockerfiles over?

**Comment by thomas-teknique** [comment]
Or additionally have a docker tag that points to the latest version that used ubuntu trusty?

**Comment by sdt** [comment]
Hi Thomas,

Sorry for the inconvenience there.

The new version is definitely worth a look. It has a proper raspbian sysroot directory and an install-raspbian script which installs raspbian packages natively into the sysroot, using qemu and chroot.

That aside, I've created a new branch called legacy-trusty, and have set up a new autobuild from it that will build sdthirlwall/raspberry-pi-cross-compiler:legacy-trusty 

The branch has been updated so that the rpxc script etc refers to the new image:tag names.

That image is building on dockerhub now. https://hub.docker.com/r/sdthirlwall/raspberry-pi-cross-compiler/builds/

Let me know how it goes.
